location/test/motif_test.py

```diff
@@ -539,3 +539,35 @@ def test_approx_home_location():
     assert utils.approx_home_location(pd.concat([df1, df2, df3])) == 'dr5xg57'
     assert utils.approx_home_location(pd.concat([df2, df3])) == 'dr5xg5g'
     assert utils.approx_home_location(df3) == 'dr5rw5u'
+
+
+def test_compute_total_gyration():
+    # test data
+    data_lons = [-73.7102671, -73.7098393, -73.71212709999998,
+                 -73.90992059999998, -73.9102825, -73.9099297,
+                 -73.9099264, -73.909925, -73.90992220000003,
+                 -73.90992490000002]
+    data_lats = [40.75110460000001, 40.7509678, 40.7523959, 40.7153243,
+                 40.7150695, 40.7153186, 40.7153194, 40.7153141,
+                 40.7153414, 40.715343]
+    data_stay_region = ['dr5xfdt', 'dr5xfdt', 'dr5xfdt', 'dr5rw5u',
+                        'dr5rw5u', 'dr5rw5u', 'dr5rw5u', 'dr5rw5u',
+                        'dr5rw5u', 'dr5rw5u']
+    data_index = ['2015-05-18 13:31:05', '2015-05-18 13:46:13',
+                  '2015-05-18 15:38:33', '2015-05-18 19:06:18',
+                  '2015-05-19 00:28:14', '2015-05-19 04:24:14',
+                  '2015-05-19 08:56:38', '2015-05-19 10:50:14',
+                  '2015-05-19 14:03:10', '2015-05-19 19:21:13']
+    df = pd.DataFrame()
+    df['latitude'] = data_lats
+    df['longitude'] = data_lons
+    df['stay_region'] = data_stay_region
+    df.index = pd.to_datetime(data_index)
+
+    # expected result
+    expected = 7935.926632803189
+
+    # tolerance = 0.01 meter
+    assert math.isclose(utils.compute_total_gyration(df),
+                        expected,
+                        abs_tol=0.01)
```

Comment:
Could you add another test here by repeating the last gps points few times (I want to make sure the duplicated points are properly calculated.